docs-site/content/guide/semantic-search.md

```diff
@@ -442,6 +442,51 @@ Notice how searching for `Desktop copier` returns `Desktop` as a result which is
 }
 ```
 
+
+### Pagination
+
+For effective pagination with semantic or hybrid search, use the `k` parameter in `vector_search` to ground the search to a specific number of closest items:
+
+```shell{11}
+curl 'http://localhost:8108/multi_search' \
+  -H "X-TYPESENSE-API-KEY: ${TYPESENSE_API_KEY}" \
+  -X POST \
+  -d '{
+    "searches": [
+      {
+        "q": "device to type things on",
+        "query_by": "embedding",
+        "collection": "products",
+        "prefix": "false",
+        "vector_query": "embedding([], k: 200)",
+        "exclude_fields": "embedding",
```

Comment:
This code example is still not helpful to understand how to do pagination. 

Could you add the `page` and `per_page` parameters to the query? Set `per_page` to may be `10` and then explain how to fetch additional pages, you'd set `page` to `1`, `2, etc.